services/121-service/src/user/user.entity.ts

```diff
@@ -5,11 +5,13 @@ import {
   BeforeInsert,
   OneToMany,
   ManyToMany,
+  JoinTable,
 } from 'typeorm';
 import { IsEmail } from 'class-validator';
-const crypto = require('crypto');
+import crypto from 'crypto';
```

Comment:
The `'crypto'`-dependency can also be removed from the `package.json`; If this was the only usage for it...

Reply:
@elwinschmitz all of this is already changed on my local machine ;)

Reply:
@elwinschmitz In general, you can assume with me that you should start reviewing only when I move the devops task to review. Otherwise it is just intermediate work.

Reply:
These comments are in a large part "notes to my future self" ;)